exo-setup/src/app-setup.spec.ls

```diff
@@ -0,0 +1,102 @@
+require! {
+  '../../exosphere-shared': {run-process}
+  'chai': {expect}
+  'fs-extra' : fs
+  'js-yaml' : yaml
+  'mkdirp'
+  'path'
+  'prelude-ls': {each}
+  'rimraf'
+}
+
+
+describe 'AppSetup', ->
+
+  describe 'set up a complex app' (...) ->
+    @timeout 600000
+
+    before (done) ~>
+      @app-name = 'complex-setup-app'
+      @app-dir = path.join process.cwd!, 'tmp', @app-name
+      @docker-compose-location = path.join @app-dir, 'tmp', 'docker-compose.yml'
+      @exocom-name = 'exocom0.22.1'
```

Comment:
Let's make this an array called `@application-dependencies` to stay consistent with the other variables.